package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ecospheres-front",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
Unrelated to this PR. Not a big deal, but let's try to make single-purpose PRs as much as possible.

Reply:
Sure, let's discuss versioning on another PR.